example/glue/configure_includes/configure_win_x86.mk

```diff
@@ -116,3 +116,17 @@ CONFIGURE_ARGS += 'OMR_HOST_OS=win'
 CONFIGURE_ARGS += 'OMR_HOST_ARCH=x86'
 CONFIGURE_ARGS += 'OMR_TARGET_DATASIZE=$(TEMP_TARGET_DATASIZE)'
 CONFIGURE_ARGS += 'OMR_TOOLCHAIN=msvc'
+
+COPYRIGHT_WIN_COMPANYNAME:=International Business Machines Corporation\0
```

Comment:
Hi Charlie, these changes (test) have not yet got verified. It seems it's got problem with initialization. Pleased hold off for review.